### DISCUSSION TITLE
9.8 support

### DISCUSSION THREAD

**Discussion by dave-re-imprivata** [rapier1/openssh-portable#88] [General]
With the release of 9.8, they've also disclosed what the OpenSSH team is rating as a critical security issue that affects 9.7 - we were just about to release 9.7 on our end, but will obviously hold due to that. Any thoughts on timeframe for 9.8 support?

**Comment by rapier1** [comment]
We've already done some of the initial patch work so we hope to have this available quickly. I'm aiming for the end of today but that might slip until Tuesday July 2.

**Comment by dave-re-imprivata** [comment]
Oh, that's perfect, Chris - thanks!

**Reply by rapier1** [reply]
9.8 is taking longer to complete than expected. With the holiday coming up we've decided to backport the fix to 9.7. This is now available on master with the tag hpn-18.4.2. 

Well get to 9.8 as soon as we can.

**Reply by dave-re-imprivata** [reply]
Thanks, Chris - the P3 patch will cover our needs for the short term. Appreciate you getting that out!
